Next version of ert will break semeio

**Describe the bug**
Ert will break module structure in next version. Building against main branch on ert gives:

```
==================================== ERRORS ====================================
_____________ ERROR collecting tests/jobs/stea/test_integration.py _____________
ImportError while importing test module '/tmp/jenkins-komodo-f_scout_ci/workspace/unit-tests/semeio/test-root/tests/jobs/stea/test_integration.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/rh/rh-python38/root/usr/lib64/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/jobs/stea/test_integration.py:8: in <module>
    from ert.shared.main import main
E   ModuleNotFoundError: No module named 'ert.shared.main'
```

**To Reproduce**
Steps to reproduce the behavior:
1. Install main on ERT
1. Run tests.

**Additional context**
Issue must be fixed **after** ERT ships new version

--- a/.gitignore
+++ b/.gitignore
@@ -22,3 +22,4 @@ core.*
 semeio/version.py
 __pycache__/
 .pytest_cache/
+.hypothesis/

--- a/tests/jobs/stea/test_integration.py
+++ b/tests/jobs/stea/test_integration.py
@@ -1,11 +1,10 @@
 """Test STEA forward model towards the HTTP STEA library
 
 """
-import sys
+import subprocess
 from pathlib import Path
 
 import pytest
-from ert.shared.main import main
 from stea import SteaKeys
 
 base_stea_config = """
@@ -41,7 +40,7 @@ ECLDIR_NORNE = Path(__file__).absolute().parent.parent.parent / "test_data" / "n
         ("a_random_ecl_case", ECLDIR_NORNE / "NORNE_ATW2013", ",<ECL_CASE>=<ECLBASE>"),
     ],
 )
-def test_stea_fm_single_real(monkeypatch, httpserver, ecl_case, ecl_base, fm_options):
+def test_stea_fm_single_real(httpserver, ecl_case, ecl_base, fm_options):
     httpserver.expect_request(
         "/foobar/api/v1/Alternative/56892/1/summary",
         query_string="ConfigurationDate=2018-11-01T00:00:00",
@@ -73,8 +72,7 @@ def test_stea_fm_single_real(monkeypatch, httpserver, ecl_case, ecl_base, fm_opt
         encoding="utf-8",
     )
 
-    monkeypatch.setattr(sys, "argv", ["ert", "test_run", "config.ert", "--verbose"])
-    main()
+    subprocess.run(["ert", "test_run", "config.ert", "--verbose"], check=True)
 
     assert Path("realization-0/iter-0/NPV_0").read_text(encoding="utf-8")[0:3] == "30\n"
     assert Path("realization-0/iter-0/stea_response.json").exists()

